sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py

```diff
@@ -17,8 +17,8 @@
 
 
 def _convert_datetime_to_utc_int(expires_on):
```

Comment:
```suggestion
def _convert_datetime_to_utc_int(input_datetime):
```

Reply:
I would prefer to keep it expires_on, which aligns with our naming in the AccessToken constructor.